AirCasting/Map/LocationTracker.swift

```diff
@@ -21,7 +21,11 @@ class LocationTracker: NSObject, ObservableObject, CLLocationManagerDelegate {
         switch locationManager.authorizationStatus {
             case .authorizedAlways, .authorizedWhenInUse:
                 self.locationGranted = .granted
-                googleLocation = [PathPoint(location: CLLocationCoordinate2D(latitude: (locationManager.location?.coordinate.latitude)!, longitude: (locationManager.location?.coordinate.longitude)!), measurement: 20)]
+                if locationManager.location?.coordinate.latitude != nil && locationManager.location?.coordinate.longitude != nil {
+                    googleLocation = [PathPoint(location: CLLocationCoordinate2D(latitude: (locationManager.location?.coordinate.latitude)!, longitude: (locationManager.location?.coordinate.longitude)!), measurement: 20)]
```

Comment:
How about using a variable here for mocked measurement? :)

Reply:
Greeeeet. Do we have some or I should create that?